openslides/utils/utils.py

```diff
@@ -29,3 +35,43 @@ def to_roman(number: int) -> str:
         return roman.toRoman(number)
     except (roman.NotIntegerError, roman.OutOfRangeError):
         return str(number)
+
+
+def get_element_id(collection_string: str, id: int) -> str:
```

Comment:
This function name is a little bit missleading because it does not return the (database) id of an element (as used above in `element.id`) but a new unique OpenSlides object identifier composed of collection name and database id.

Proposal: Rename it to `get_element_uid()` for element unique identifier.

Reply:
I see what you mean. I don't like uid. When I here it, I think of an user id. See also https://de.wikipedia.org/wiki/UID

I use the term 'element_id' at a lot of places. See the file cache.py. I thought element_id is ok, because we did not use this term in the code before.